### ISSUE TITLE
Improved Variable List organization

### ISSUE THREAD

**Issue by tatiang** [thunkable/thunkable-issues#740]
**Is your feature request related to a problem? Please describe.**
I've always been a little confused about how variables are sorted in the Blocks tab. It seems like they might be in alphabetical order or chronological order (as created) or by type... it's never clear to me.

**Describe the solution you'd like**
I can think of a few improvements: (1) a text list that's accessible somewhere in addition to the blocks; not quite sure how that would work or how you'd modify variable names (2) the re-design example shown below, where variable types have section headers.

**Describe alternatives you've considered**
Would be great to have someone explain the current sort method. 👍 

**Which platform are you asking for?**
Thunkable ✕ (x.thunkable.com)

**Additional context**
![Variable List redesign](https://user-images.githubusercontent.com/10355498/99019560-6b4fdd80-2511-11eb-90ee-a2983d717edb.png)

**Comment by DrTedGit** [comment]
Agreed. The chronological order is fine for small apps built in one session. For apps built and maintained over months and years, chronological order is a real challenge

**Comment by tatiang** [comment]
I also think a lot can be done with color differences, icons, etc. to differentiate between variable types without using text ("app", "stored", "cloud" right within the block name).
